installer/templates/new/widgets/text/text.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import Widget from '../../assets/javascripts/widget';
+import {updatedAt} from '../../assets/javascripts/helpers';
 
 import './text.scss';
 
 Widget.mount(class Text extends Widget {
   render() {
     return (
-      <div className={this.state.classname || this.props.className}>
+      <div className={this.props.className + " " + this.state.specialStyle}>
         <h1 className="title">{this.state.title || this.props.title}</h1>
         <h3>{this.state.text || this.props.text}</h3>
         <p className="more-info">{this.state.moreinfo || this.props.moreinfo}</p>
+        <p className="updated-at">{updatedAt(this.state.updated_at)}</p>
```

Comment:
I noticed that the stylesheet had this class but the widget did not add it as a field. I kind of think most widgets will want to have last updated so I put it in.